internal/init/init.go

```diff
@@ -132,18 +135,19 @@ func (c *Client) NewCommand() *cobra.Command {
 	return initCmd
 }
 
-func getFirstProjectID(s packngo.ProjectService, userOrg string) (string, error) {
-	listOpts := &packngo.ListOptions{}
-	listOpts.Including("organization")
-	listOpts.Excluding("devices", "members", "memberships", "invitations", "max_devices", "ssh_keys", "volumes", "backend_transfer_enabled", "updated_at", "customdata", "event_alert_configuration")
+func getFirstProjectID(s metal.ProjectsApiService, userOrg string) (string, error) {
+	include := []string{"organization"} // []string | Nested attributes to include. Included objects will return their full attributes. Attribute names can be dotted (up to 3 levels) to included deeply nested objects. (optional)
+	exclude := []string{"devices", "members", "memberships", "invitations", "ssh_keys", "volumes", "backend_transfer_enabled", "updated_at", "customdata", "event_alert_configuration"}
+	page := int32(56)    // int32 | Page to return (optional) (default to 1)
+	perPage := int32(56) // int32 | Items returned per page (optional) (default to 10)
 
-	projects, _, err := s.List(listOpts)
+	projects, _, err := s.FindProjects(context.Background()).Include(include).Exclude(exclude).Page(page).PerPage(perPage).Execute()
 	if err != nil {
 		return "", err
 	}
-	for _, p := range projects {
-		if p.Organization.ID == userOrg {
-			return p.ID, nil
+	for _, p := range projects.GetProjects() {
+		if p.AdditionalProperties["organization"] == userOrg {
```

Comment:
`p.AdditionalProperties["organization"]` will be an object/map;  you'll probably need to do something more like:
```
organization := p.AdditionalProperties["organization"].(map[string]interface{})
organization_id := fmt.Sprintf("%v", organization["id"])
```

Reply:
updated